platform/src/pages/analytics/tabs/OverView.jsx

```diff
@@ -1,15 +1,85 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import ChartContainer from '@/components/Charts/ChartContainer';
+import AQNumberCard from '@/components/AQNumberCard';
+import BorderlessContentBox from '@/components/Layout/borderless_content_box';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRecentMeasurementsData } from '@/lib/store/services/deviceRegistry/RecentMeasurementsSlice';
 
 const OverView = () => {
+  // events hook
+  const dispatch = useDispatch();
+  const recentLocationMeasurements = useSelector((state) => state.recentMeasurements.measurements);
+  const chartDataRange = useSelector((state) => state.chart.chartDataRange);
+  const userDefaults = useSelector((state) => state.userDefaults.defaults);
+  const [grids, setGrids] = useState([
+    '64b7baccf2b99f00296acd59',
```

Comment:
Hi @Codebmk , thanks for this. Can we set these as variables? Somewhere in a constants file? This is just good practice. Please note that we do **NOT** have to create secrets or ENVs, we just need to put all constants in one place ---- as a best practice.

Reply:
done